Pipfile

```diff
@@ -4,7 +4,7 @@ verify_ssl = true
 name = "pypi"
 
 [dev-packages]
-
```

Comment:
I think we should add `pytest = "*"` as `pytest` is installed with we install the requirements!